### PR TITLE
Refactor install

### PR DESCRIPTION
**Pull request by colin-higgins**
- Installer log is consolidated in datadog log files directory
 - Remove logging logic from `install.cmd`
 - Implement https://github.com/DataDog/datadog-aas-extension/issues/48
 - Better logging for diagnosing install problems

### CODE REVIEW

File: dotnet/content/scmApplicationHost.xdt [hunk 1]
```diff
@@ -2,4 +2,16 @@
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
   <!-- This file exists to prevent applicationHost.xdt from being applied to scm host that runs
       other dotnet processes (dotnet build, csc etc. )-->
+	<environmentVariables xdt:Transform="InsertIfMissing">
+        
+        <add name="DD_APM_WINDOWS_PIPE_NAME" value="datadogtrace-uniqueTracePipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/>
```
Comment (zacharycmontoya) [inline dotnet/content/scmApplicationHost.xdt:7]: Do you need to do the thing where you have `xdt:Transform="RemoveAll"` followed by `xdt:Transform="Insert"`?

**Review by zacharycmontoya — APPROVED** [review]

File: dotnet/content/install.ps1 [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
-    [Parameter(Mandatory=$true)][string]$ExtensionVersion
-)
-
 # https://github.com/projectkudu/kudu/blob/13824205c60a4bdb53896b9553ef1f370a93b911/Kudu.Services/SiteExtensions/SiteExtensionController.cs#L240
 
+$Version="vUNKNOWN"
+
+function Log ([string] $text)
+{
+    $LogDate=Get-Date -Format "MM/dd/yyyy HH:mm K"
```
Comment (shurivich) [inline dotnet/content/install.ps1:7]: Just a suggestion about format, I think "sortable" suits better for logs?
```suggestion
    $LogDate=Get-Date -Format "yyyy-MM-dd HH:mm K"
```

**Review by shurivich — APPROVED** [review]

File: dotnet/content/scmApplicationHost.xdt (hunk 1 above)
Comment (colin-higgins) [reply dotnet/content/scmApplicationHost.xdt:7]: Not needed, but it's good practice and consistent.
I think I will re-org the application host to be more readable in another PR as well.
There are also some things we may not want to override if the user sets them.
Like disabling the profiler.

**Review by lucaspimentel — APPROVED** [review]

File: dotnet/content/install.ps1 (hunk 2 above)
Comment (colin-higgins) [reply dotnet/content/install.ps1]: Done

File: dotnet/content/scmApplicationHost.xdt (hunk 1 above)
Comment (colin-higgins) [reply dotnet/content/scmApplicationHost.xdt]: Done

**Pull request merged**